Clicker/Quarks/PollQuark.swift

```diff
@@ -94,12 +94,12 @@ struct GetSortedPolls: ClickerQuark {
                     let pollsArr: [Poll] = pollsArray.reversed().map {
                         return PollParser.parseItem(json: $0)
                     }
-                    let pollsDateModel = PollsDateModel(date: date, polls: pollsArr)
+                    let pollsDateModel = PollsDateModel(date: date.trimmingCharacters(in: .whitespaces), polls: pollsArr)
                     pollsDateArray.append(pollsDateModel)
                 }
             }
             let formatter = DateFormatter()
-            formatter.dateFormat = "dd/MM/yyyy"
+            formatter.dateFormat = "MMM dd yyyy"
```

Comment:
Make a constant for the string `"MM dd yyyy"`, since we use it in a few different places.